tests/main/interfaces-cups/task.yaml

```diff
@@ -0,0 +1,66 @@
+summary: Ensure that the cups interfaces work with app providers
+
+details: |
+    A snap providing the cups-control and cups interfaces should be able to
+    create the control socket, with connecting consuming snaps able to use it.
+    This intentionally does not test the mediation properties of the cupsd
+    server.
+
+prepare: |
+    snap pack test-snapd-provider
+    snap install --dangerous ./test-snapd-provider_1_all.snap
```

Comment:
minor nitpick, I think this can be simplified to just `install_local test-snapd-provider`